### DISCUSSION TITLE
Issue with Verifying Framework in ShadCN Setup (React + Rspack)

### DISCUSSION THREAD

**Discussion by huyixi** [web-infra-dev/rsbuild#3602] [Q&A]
I’m currently trying to set up ShadCN in my project(Rsbuild + React + shadcnui), but I’m encountering an issue during the “Verifying framework” step. The error message I get is:

➜ nlx shadcn@latest add button
✔ You need to create a components.json file to add components. Proceed? … yes

We could not detect a supported framework at /Users/huyixi/i/store-app-demo/frontend.
Visit https://ui.shadcn.com/docs/installation/manual to manually configure your project.
Once configured, you can use the cli to add components.

**Comment by chenjiahan** [comment]
This should not be related to Rsbuild, you can report it to shadcn/ui.
